co-simulation/fed/mosaic-infrastructure/src/main/java/org/eclipse/mosaic/fed/infrastructure/ambassador/V2xInstance.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2023 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+ 
+package org.eclipse.mosaic.fed.infrastructure.ambassador;
+
+import org.eclipse.mosaic.lib.geo.GeoPoint;
+
+import java.io.IOException;
+import java.net.DatagramPacket;
+import java.net.DatagramSocket;
+import java.net.InetAddress;
+import java.net.SocketException;
+
+/**
+ * Connection manager and data object to associate with a single infrastructure instance in XIL
+ * NOTE: TODO See carma.ambassador for reference
+ */
+public class V2xInstance {
+   
+    private DatagramSocket rxMsgsSocket = null;
+    private InetAddress targetAddress;
+    private int port;
+    private GeoPoint location = null;
+
+    public V2xInstance() {
+        // TODO
```

Comment:
Add to comment : TODO Initialize Datagram Socket

Reply:
Done